app/controllers/forms_controller.rb

```diff
@@ -31,7 +31,6 @@ def mark_pages_section_completed
     if @mark_complete_form.mark_section
       redirect_to form_path(@form)
     else
-      flash[:message] = "Save unsuccessful"
       render "pages/index", status: :unprocessable_entity
```

Comment:
I wonder if its worth "resetting" the radio button. Currently if there are validation messages it will still show the "Yes" selected for completed when it hasn't actually been saved.

Reply:
Think this is a good idea, have implemented